### PR TITLE
Create Github templates

### PR DESCRIPTION
**Pull request by tai271828**
By using the templates we could guide people to provide necessary
information in the expected way, so we may reduce the cost of
communication back and forth.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+name: 📖 Documentation
+about: Suggest an improvement for the documentation of this project
+title: "[Documentation] Content to be added or fixed"
+labels: "documentation"
+assignees: ""
+---
+
+## Type
+* [ ] Conent inaccurate
```
Comment (dannf) [inline .github/ISSUE_TEMPLATE/documentation.md]: s/Conent/Content/

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+title: "[Bug Report] Good bug title tells us about precise symptom, not about the root cause."
+labels: "bug"
+assignees: ""
+---
+
+## Description
+<!-- A clear and concise description of what the bug is. -->
+
+## {{ cookiecutter.project_name }} version
+<!-- x.y.z -->
```
Comment (dannf) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Note that there currently isn't any released versions (not sure there ever will be, given the nature of this script)

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+1. ...
+2. ...
+3. ...
+-->
+
+## Expected behavior
+<!--A clear and concise description of what you expected to happen-->
+
+## Related Issue
+<!--If applicable, refernce to the issue related to this pull request.-->
```
Comment (dannf) [inline .github/pull_request_template.md]: s/refernce/reference/

**Review by dannf — COMMENTED** [review]
Thanks @tai271828 ! This does look helpful. Are these templates pulled from an existing location, or something you created? If from somewhere else, could you note that in the commit?

**Comment by tai271828** [comment]
> Thanks @tai271828 ! This does look helpful. Are these templates pulled from an existing location, or something you created? If from somewhere else, could you note that in the commit?

Thanks for review and the reminder. The templates were developed by my friends and me for some of our side projects. The template evolve organically when time goes by for years. I will double check if it has any license issue.

**Comment by tai271828** [comment]
> > Thanks @tai271828 ! This does look helpful. Are these templates pulled from an existing location, or something you created? If from somewhere else, could you note that in the commit?
> 
> Thanks for review and the reminder. The templates were developed by my friends and me for some of our side projects. The template evolve organically when time goes by for years. I will double check if it has any license issue.

The templates are collected in [this "cookiecutter" templates for python](https://github.com/Lee-W/cookiecutter-python-template) maintained by @Lee-W . Cookiecutter is a project template generator, and users could generate their own projects with their own customized templates if they want. If we use [this "cookiecutter" templates for python](https://github.com/Lee-W/cookiecutter-python-template) maintained by @Lee-W to generate a python project, we will get the same template content. The template is MIT license so I think we could feel free to use. I will add the information of the template source in the fixup of this PR.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (tai271828) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Let me add comments like "x.y.z or commit hash". Thanks for the comment!

**Comment by tai271828** [comment]
All fix suggestions were resolved, including the update of README to show the original source of github template.

**Pull request merged**